### DISCUSSION TITLE
1xBET statistics bot for BACCARAT

### DISCUSSION THREAD

**Discussion by MonstroAz** [community/community#113409] [Programming Help]
### Select Topic Area

General

### Body

I want to make a statistics bot for baccara like this one
![image](https://github.com/community/community/assets/20817411/55f8ab26-8d11-47d1-ad43-394ee1bb7b1c)

im using chat gpt to help me, right now i scraped a bit the 1xbet website and got some informations, now i dont know that to do, 
[test 1.json](https://github.com/community/community/files/14664250/test.1.json) ended up with this code

IMPORTANT INFO
![image](https://github.com/community/community/assets/20817411/40a88a3a-e101-41f7-a84f-50d36230438f)
THIS IS ALL THE DATA FROM 1 GAME

DI: game number
this section shows the status of the game (player and banker total points, and game status):
  "CPS": "",
                                "FS": {
                                    "S1": 7,
                                    "S2": 7
                                },
                                "I": "Game finished",

this one is the outcome:

  "S": [
player cards (card numbers and suits)                                   
 {
                                        "Key": "P",
                                        "Value": "[{\"S\":3,\"R\":4},{\"S\":1,\"R\":8},{\"S\":2,\"R\":5}]"
                                    },
banker...
                                    {
                                        "Key": "B",
                                        "Value": "[{\"S\":2,\"R\":5},{\"S\":1,\"R\":2}]"
                                    },
outcome:
                                    {
                                        "Key": "S",
                                        "Value": "Draw"
                                    }
                                ],

Im very  noob, and learning everything today only, i need to make a bot on TG that will send this data to my channel on this format #N570. 8(Q♥️8♦️) - 2(2♥️10♣️)

anny help??

**Comment by MonstroAz** [comment]
[test99999.json](https://github.com/community/community/files/14666782/test99999.json)
![Screenshot 2024-03-20 165608](https://github.com/community/community/assets/20817411/48c17e6a-ce99-48bf-81bb-631944c45980)
![Screenshot 2024-03-20 165547](https://github.com/community/community/assets/20817411/dee7baeb-74c5-4857-ba9a-8b37e16d6b36)
![Screenshot 2024-03-20 165520](https://github.com/community/community/assets/20817411/cee2771c-0be2-4d83-bd44-30d37d557bdc)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MonstroAz ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by Shahseo** [comment]
ok

**Comment by scofield324** [comment]
hi mate send me ur telegram
